BaseApi.js

```diff
@@ -144,6 +145,9 @@ const API = {
     const url = module_url.replace('$ideaPk', ideaPk);
     return makePutRequest(url, data, token);
   },
+  postReport(content_type, object_pk, description, token = null) {
```

Comment:
maybe replace the curly braces just with `data`  here and add the curly braces in the `handleSubmit` so it's more in style with the other functions here?